src/Build/Evaluation/Profiler/ProfilerResultPrettyPrinter.cs

```diff
@@ -2,35 +2,43 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //-----------------------------------------------------------------------
 // </copyright>
-// <summary>Transforms a profiled result to markdown form.</summary>
+// <summary>Pretty prints a profiled result to a variety of formats.</summary>
 //-----------------------------------------------------------------------
 
 using System;
-using System.Collections.Concurrent;
-using System.Collections.Generic;
 using System.Diagnostics;
-using System.IO;
 using System.Linq;
 using System.Text;
-using Microsoft.Build.Construction;
 using Microsoft.Build.Framework.Profiler;
 
 namespace Microsoft.Build.Evaluation
 {
     /// <summary>
-    /// Transforms a profiled result to markdown form
+    /// Pretty prints a profiled result to a variety of formats
```

Comment:
Since the logger is added via a command switch, can you please make it internal?

Reply:
Nwm, thought this was the logger :)

---
In reply to: [159565499](https://github.com/Microsoft/msbuild/pull/2837#discussion_r159565499) [](ancestors = 159565499)